### DISCUSSION TITLE
Custom Light Levels

### DISCUSSION THREAD

**Discussion by 2001herne** [SchmarrnDevs/Lighty#20] [Ideas and Suggestions]
I'm thinking it might be nice to be able to set our own thresholds for the light levels - while most players will be fine with the pure spawnable light levels, some may prefer to use their own thresholds as to what is considered an 'acceptable' light level (i.e. some may prefer to keep to the old light level of > 8)

**Comment by andi-makes** [comment]
Thank you for your feedback!

Funnily enough, I've already implemented that feature for the snapshots and 1.19.4 - I just hadn't time to port it to 1.19.2 and release it because I've been very busy with uni stuff.

But I should have time to do the porting and release the feature today, I'll comment here when it's done!

**Comment by andi-makes** [comment]
[You can find the new version on Modrinth now](https://modrinth.com/mod/lighty) - version 2.0.0-alpha.3 for the most recent 1.20 snapshot, and version 1.3.0 for minecraft 1.19.2 and 1.19.4 - have fun!
